lib/msf/core/session_manager.rb

```diff
@@ -20,7 +20,7 @@ class SessionManager < Hash
 
   include Framework::Offspring
 
-  LAST_SEEN_INTERVAL = 60 * 2.5
+  LAST_SEEN_INTERVAL  = 60 * 2.5
```

Comment:
Minor: extra space added

Reply:
LOL I literally did not see this comment when I made mine.